golang 1.15 generates a compile error.

### What happened?

Pulled provider-azure to hack on a bug, ran make, got an error about unsupported go version; I have 1.15.6

### How can we reproduce it?
Use go 1.15.6 to build use the Makefile.

### What environment did it happen in?
```
make
16:03:03 [ .. ] verify dependencies have expected content
all modules verified
16:03:05 [ OK ] go modules dependencies verified
16:03:05 [FAIL] unsupported go version. Please make install one of the following supported version: 1.7|1.8|1.9|1.10|1.11|1.12|1.13|1.14
make[2]: *** [build/makelib/golang.mk:147: go.init] Error 1
make[1]: *** [build/makelib/common.mk:302: build.all] Error 2
make: *** [build/makelib/common.mk:313: build] Error 2
robertc@robertc-linux:~/src/provider-azure$ go version
go version go1.15.6 linux/amd64
robertc@robertc-linux:~/src/provider-azure$ git describe
v0.15.0-rc.0
```

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -34,6 +34,14 @@ updated. After initial setup, it can be updated by running `make submodules`.
 
 Run `make help` for more options.
 
+### Go support
+
+To build with as-yet unvalidated go versions set GO_SUPPORTED_VERSIONS:
+
+```console
+GO_SUPPORTED_VERSIONS=1.15 make
+```
+
 ## Building inside the cross container
 
 Official Crossplane builds are done inside a build container. This ensures that we get a consistent build, test and release environment. To run the build inside the cross container run:

